apps/web/src/server/lib/nunjucks-filters/trim.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Trim whitespace from both ends of a string and return a new string without modifying the original.
+ *
+ * @param {string} stringToTrim – The string to trim.
+ * @returns {string} - The trimmed string
+ */
+export function trim(stringToTrim) {
+	return stringToTrim.trim();
+}
```

Comment:
I might have missed something but just wondering how this custom filter differs from the built in one (https://mozilla.github.io/nunjucks/templating.html#trim)?